code/game/objects/structures/customplaque.dm

```diff
@@ -0,0 +1,146 @@
+/obj/structure/customplaque //This is a plaque you can craft with gold, then permanently engrave a title and description on, with a fountain pen.
+	icon = 'icons/obj/decals.dmi'
+	icon_state = "blankplaque"
+	name = "blank plaque"
+	desc = "A blank plaque, use a fancy pen to engrave it. It can be detatched from the wall with a wrench."
+	anchored = TRUE
+	opacity = 0
+	density = FALSE
+	layer = SIGN_LAYER
+	custom_materials = list(/datum/material/gold = 2000)
+	max_integrity = 200 //Twice as durable as regular signs.
+	armor = list("melee" = 50, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
+	rad_flags = RAD_PROTECT_CONTENTS | RAD_NO_CONTAMINATE
+	var/unengraved = TRUE //Custom plaque structures and items both start "unengraved", once engraved with a fountain pen their text can't be altered again.
+
+/obj/item/customplaque
+	icon = 'icons/obj/decals.dmi'
+	icon_state = "blankplaque"
+	name = "blank plaque"
+	desc = "A blank plaque, use a fancy pen to engrave it. It can be placed on a wall."
+	w_class = WEIGHT_CLASS_NORMAL
+	custom_materials = list(/datum/material/gold = 2000)
+	max_integrity = 200
+	armor = list("melee" = 50, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
+	var/plaque_path = /obj/structure/customplaque //This points the item to make the proper structure when placed on a wall.
+	var/unengraved = TRUE
+
+/obj/structure/customplaque/play_attack_sound(damage_amount, damage_type = BRUTE, damage_flag = NONE)
+	switch(damage_type)
+		if(BRUTE)
+			if(damage_amount)
+				playsound(loc, 'sound/weapons/smash.ogg', 80, TRUE)
+			else
+				playsound(loc, 'sound/weapons/tap.ogg', 50, TRUE)
+		if(BURN)
+			playsound(loc, 'sound/items/welder.ogg', 80, TRUE)
```

Comment:
is this not handled elsewhere already?

Reply:
I have absolutely no idea.

Reply:
it is.

Reply:
I can just remove the proc completely then?

Reply:
I believe I fixed this. Signs now make a slash noise when hit but I got rid of the rest of the code there.

Reply:
I just axed the slash sound completely, it's kind of out of place for a plastic sign.